Mastermind/Mastermind.cs

```diff
@@ -1,86 +1,143 @@
 ﻿using System;
 
-namespace Mastermind
+public class Program
 {
-    class Program
+    // possible letters in code
```

Comment:
You did not really used classes, and this assignment is supposed to be using object oriented programing.